apps/coscrad-frontend/src/utils/generic-components/presenters/multiple-property-presenter.tsx

```diff
@@ -18,15 +23,20 @@ export const MultiplePropertyPresenter = <T,>({
         <>
             {Object.entries(keysAndLabels)
                 .filter(([propertyKey, _]) => !isNullOrUndefined(data[propertyKey]))
-                .map(([propertyKey, label]) => {
-                    const propertyValue = data[propertyKey];
-
-                    if (!isString(propertyValue) && !isNumber(propertyValue)) {
+                .filter(([propertyKey, _]) => {
```

Comment:
Umm, @aaron-plahn , not sure you're going to like this.  I'm trying to prevent property labels (e.g., 'Contributors:') from displaying with no associated value.  This would be true for a citation with missing/unfindable information from the original source.